src/components/Headline/Headline.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+Headline.propTypes = {
+  childre: PropTypes.node.isRequired,
+  small: PropTypes.bool,
+}
+
+export default function Headline({ children, small }) {
```

Comment:
I suggest calling the function heading instead of headline

Reply:
Thank you for your review :)
Yes a very good point. I'm changing that.